core/templates/dev/head/pages/exploration-editor-page/editor-navigation/editor-navigation-dropdown.directive.html

```diff
@@ -0,0 +1,164 @@
+<style type="text/css">
+  .oppia-editor-dropdown {
+    position: absolute;
+    right: 100px;
+    top: 10px;
+  }
+  .oppia-editor-dropdown .oppia-dropdown-element {
+    padding-left: 12px;
+  }
+  .oppia-editor-dropdown .oppia-dropdown-element .oppia-dropdown-inner{
+    padding-left: 10px;
+  }
+</style>
+<div class="dropdown hidden-lg hidden-md ng-cloak oppia-editor-dropdown">
+  <a class="btn dropdown-toggle" style="background-color: #009b8c; color: #ffffff;" type="button" data-toggle="dropdown">Options
+    <span class="caret"></span>
+  </a>
+  <ul class="dropdown-menu" style="min-width: 100px">
```

Comment:
As mentioned before, please get rid of inline styles and use classes throughout. Ditto elsewhere.

Reply:
I am sorry I thought since inline styling were used in other files so I thought that I could also use it.
Now I have removed every inline styling.